app/app/src/main/kotlin/com/hedvig/app/feature/profile/ui/payment/PaymentDestination.kt

```diff
@@ -347,14 +382,49 @@ fun PaymentDetails(uiState: PaymentUiState) {
   }
 }
 
+@Composable
+fun PayoutDetails(uiState: PaymentUiState) {
+  Text(
+    text = stringResource(id = R.string.payment_screen_payout_section_title),
+    modifier = Modifier.padding(horizontal = 16.dp),
+  )
+  Spacer(Modifier.height(16.dp))
+  Divider(modifier = Modifier.padding(horizontal = 16.dp))
+  Spacer(Modifier.height(4.dp))
+  Row(
+    modifier = Modifier
+      .fillMaxWidth()
+      .padding(vertical = 12.dp, horizontal = 16.dp),
+    horizontalArrangement = Arrangement.SpaceBetween,
+    verticalAlignment = Alignment.CenterVertically,
+  ) {
+    Row(
+      verticalAlignment = Alignment.CenterVertically,
+    ) {
```

Comment:
There seems to maybe be an extra Row in here that is not necessary? Maybe it was here before for the chevron or something like that, but if it's so I think you can just delete it and it should work.